src/komponenter/delseksjon.tsx

```diff
@@ -39,55 +37,45 @@ export function Delseksjon({ delseksjon, delseksjonIndeks }: seksjonProps) {
         skalAvsnittInkluderesState,
         skalAvsnittInkluderesDispatch,
     } = useContext(SkjemaContext);
+    const { mellomlagringDelseksjonerDispatch, mellomlagringDelseksjonerState } =
+        React.useContext(MellomlagringContext);
     const [fritekstTabell, settFritekstTabell] = useState<string[][]>([]);
     const [fritekst, settFritekst] = useState<string>('');
-    const [flettefelt, settFlettefelt] = useState<SanityChildren[][]>([]);
+    const [flettefelt, settFlettefelt] = useState<flettefelt[][]>([]);
 
     useEffect(() => {
         settFritekst('');
         if (delseksjon.innhold !== null) {
-            const { nyFritekstTabell, nyFlettefelttabell } =
-                innholdTilFritekstTabellOgFlettefeltTabell(delseksjon.innhold);
+            const nyFritekstTabell = innholdTilFritekstTabell(delseksjon.innhold);
+            const nyFlettefeltTabell = innholdTilFlettefeltTabell(
+                delseksjon.innhold,
+                mellomlagringDelseksjonerState[delseksjonIndeks].innhold
+            );
 
             settFritekstTabell(nyFritekstTabell);
```

Comment:
Friteksttabell må oppdateres med mellomlagret avsnitt for å fungere riktig. Fritekstabell settes nå til å være tom, og ved endringer vil følgende skje: 

- Hvis man velger i dropdown før man trykker i fritekstfeltet vil dette oppdatere et felt i friteksttabellen. Så vil hele friteksttabellen gjøres om til streng for å vises. Siden resterende elementer i tabellen er tomme vil resten av innholdet i friteksten slettes. 
- Hvis man trykker i fritekstfeltet med en gang og så gjør endring i dropdown fjernes alt som var der og man får kun det som akkurat ble valgt